example/centos/fragment/fix-bls/default.yaml

```diff
@@ -0,0 +1,2 @@
+type: org.osbuild.fix-bls
+options: {}
```

Comment:
I think it would be good if both of these fragments have a comment that point to https://github.com/osbuild/images/blob/v0.90.0/pkg/manifest/os.go#L439 and explains a little bit what the rules for adding a BLS stage are (and also it's one thing to think about, i.e. here it is nice that images is code as it can make this decision automatically (cc @ondrejbudai @achilleas-k ) - it would be nice to think if there is a way to provide some user safeguards (an external seems overkill but might work)

Reply:
It seems to me that perhaps the creation of the fix-bls stage could be handled by the partition external? That one knows if `/boot` is on a (separate) partition.

Reply:
I like the idea of re-using the partition external

Reply:
https://github.com/osbuild/otk/issues/252